src/settings.ts

```diff
@@ -441,6 +451,19 @@ export const ROOM_TEMPLATES: RoomTemplate[] = [
       },
     ],
   },
+  {
+    template: VenueTemplate.embeddable,
+    name: "Embeddable",
+    description: "Add an embeddable with an embedded iframe in the center",
```

Comment:
[NB] We could probably come up with some better words to use here.

Reply:
fixed in c95c932f2c2b10cfa98f09884f67ddfd551ced42 accroding to the product team in thread https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1615913489012800